contrib/checkbox-provider-ce-oem/units/strict-confinement/mediacard-strict-confinement.pxu

```diff
@@ -0,0 +1,67 @@
+plugin: user-interact
+template-engine: jinja2
```

Comment:
Why do you use `template-engine: jinja2` here? I cannot find any requirement in this job.

Reply:
That is a redundant one. Remove it already. Thanks for the review.